docs/content/recipes/interoperability/identifier-mapping.md

```diff
@@ -0,0 +1,384 @@
+       class IE select;
+       class SL select;
+      %% style IM fill:#f9f
+      %% style I fill:#f9f
+      %% style OxO fill:#f9f
+```
+___
+
+**ToDo:** Update diagram
+
+
+<div class="mermaid">
+graph TD
+
+  Process2[Identifier Minting]:::bix --> A1(authority decide identity):::box
+  Process2 --> Process3
+  Process3[URI construction]:::bix --> A2(scoping the authority):::box
+  Process3 --> Process4
+  Process4[URI resolution]:::bix --> A3(directing requests to the relevant identified entity):::box
+
+  classDef bix font-family:avenir,font-size:14px,fill:#7e0038,stroke:#222,color:#fff,stroke-width:1px
+  classDef box font-family:avenir,font-size:14px,fill:#2a9fc9,stroke:#222,color:#fff,stroke-width:1px
+</div>
+
+---
+
+
+## Capability & Maturity Table
+
+| Capability  | Initial Maturity Level | Final Maturity Level  |
+| :------------- | :------------- | :------------- |
+| Interoperability | minimal | repeatable |
+| Identifier mapping ||repeatable|
+
+----
+<!-- 
+## FAIRification Objectives, Inputs and Outputs
+
+| Actions.Objectives.Tasks  | Input | Output  |
+| :------------- | :------------- | :------------- |
+| [Identifier](http://edamontology.org/data_0842) |   |   |
+ -->
+
+## Table of Data Standards
+
+| Data Formats  | Terminologies | Models  |
+| :------------- | :------------- | :------------- |
+| [IRI](https://tools.ietf.org/html/rfc3987) |   |   |
+| [CURIE](https://www.w3.org/TR/2010/NOTE-curie-20101216/) |   |   |
+| [URL](https://tools.ietf.org/html/rfc1738) |  |  |
+| [HTTP]() |  |  |
+| [RDF]() |  |  |
+| [CSV]() |  |  |
+| [TSV]() |  |  |
+| [VoID Linkset]() |  |  |
+
+---
+## Mappings
+Before diving into identifier mapping it is important to understand the possible types of mappings that can be performed between entities. While initially we might think of mapping as simply linking identical entities in different databases/formats, sometimes related entities might also be of interest. 
+When these mappings might only be interesting depending on the context in which data is being used, we run into a situation that has been described as "scientific lenses". These lenses allow us
+to dynamically select which mappings to consider relevant and which to ignore. For example allowing or disallowing mappings between stereoisomers or between genes and proteins.
+
+Examples of types of mappings are:
+* **Content mapping**: where we are mapping the actual entities by using techniques such as BLAST in biological sequences or comparison of INCHI identifiers for chemical compounds
+* **Ontology mapping**: this can either be 
+    * As a direct 1-to-1 mapping between equivalent terms in different ontologies
+    * As a complex m-to-m mapping between terms in different ontologies taking into account their hierarchical structure, see [[2]](#References).
+* **Identifier mapping**: The focus of this relationship. This can either be:
+    * Mapping between differently formed identifiers that resolve to the same entity.
```

Comment:
add an example here

Reply:
Do these examples make sense to you?

* e.g. mapping the ontology term for gene in GO and HGNC
* e.g. mapping a gene to all its possible gene products 
* e.g. mapping a gene from HGNC to the same gene in Ensembl